stdlib/http/src/main/ballerina/src/http/resiliency/http_retry_client.bal

```diff
@@ -69,7 +69,7 @@ public type RetryClient client object {
     # + message - An HTTP outbound request message or any payload of type `string`, `xml`, `json`, `byte[]`,
     #             `io:ReadableByteChannel` or `mime:Entity[]`
     # + return - The HTTP `Response` message, or an error if the invocation fails
-    public remote function post(string path, RequestMessage message) returns Response|ClientError {
+    public remote function post(string path, RequestMessage message) returns @tainted Response|ClientError {
         var result = performRetryAction(path, <Request>message, HTTP_POST, self);
```

Comment:
Don't we need to sanitize the return of the `performRetryAction` method here and return the sanitized output?
Because IMO the taintedness should not be passed to the end-user who uses the HTTP client actions with retry config.

Reply:
I don't see how we can sanitize the values in a general way given that they can contain arbitrary data

Reply:
Yes. Agree with that. IMO, we need to fix https://github.com/ballerina-platform/ballerina-lang/issues/19844.